docs/workflow_tasks/move_granules.md

```diff
@@ -0,0 +1,57 @@
+---
+id: move_granules
+title: Move Granules
+hide_title: true
+---
+
+# Move Granules
+
+This task utilizes the Cumulus Message Adapter to interpret and construct incoming and outgoing messages.
+
+Links to the npm package, task input, output and configuration schema definitions and more can be found on the auto-generated [Cumulus Tasks](../tasks) page.
+
+## Summary
+
+This task utilizes the incoming config.input_granules and the task input list of s3 URIs along with the rest of the configuration objects to do the following for a list of files assigned to a single collection:
+
+* Take the list of incoming files and sort them into a list of granule objects.  Assign files to existing granules where appropriate.
+
+* Move granules from their 'staging' location to the final location (as configured )
```

Comment:
There is an extra space after `as configured`, but I would change this to (as configured in the Sync Granules task)

Reply:
62e4f3ca1